.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "printWidth": 120,
+  "singleQuote": true
```

Comment:
I know this will evolve, but can we add the following rules already:

“trailingComma”: “all”,
 “tabWidth”: 2, 
 “semi”: true

Reply:
I didn't specify them because they are the default value (at least for `tabWidth` and `semi`), but I can make it explicit, this way we know what we have 🙂